openedx/adg/lms/webinars/helpers.py

```diff
@@ -212,6 +213,19 @@ def remove_emails_duplicate_in_other_list(email_list, reference_email_list):
     return [email for email in email_list if email not in reference_email_list]
 
 
+def is_webinar_upcoming(webinar):
```

Comment:
How about making it a property of the Model? as it is directly associated with a single instance of the model. And if we have a look at the usages of this helper, my point would create more sense